Samples~/Waypoint Example/WaypointExample.cs

```diff
@@ -160,4 +160,28 @@ private void AtiaBlessing()
         var data = ABI.EncodeFunctionData(readableAbi, values);
         Execute("Atia Blessing", () => Waypoint.OnCallContract(atiaShrineContractAddress, data));
     }
+
+    [UnityEditor.MenuItem("MyMenu/Encode Swap abis")]
+    static void EncodeSwapAbi()
```

Comment:
[MUST] Debug method must not be included in sample code.

Reply:
Solve